src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -103,7 +103,7 @@ public ArgumentProcessor(String[] args, Mode startupMode, PreferencesService pre
             ParserResult result = new ParserResult(entries);
             result.setToOpenTab();
             return Optional.of(result);
-        } catch (ParseException e) {
+        } catch (ParseException | IOException e) {
```

Comment:
Please do not add the IOExceptio here

Reply:
> Please do not add the IOExceptio here

Sure I will remove the exceptions that I added, and move the method you mentioned.
I have a question, what do you think more attributes should be added to the resulting bibtex, the metadata of the API returns much more information, I looked at other fetchers and added the common attributes for a start, you can tell me if you would like other attributes to be added, for me I think a description should be added.

Reply:
You should import all fields that can somehow be mapped to the BibTeX/biblatex
e.g. abstract (that is very useful) language, location, and maybe add the terms under other as keywords?